apps/dashboard/app/views/projects/show.html.erb

```diff
@@ -33,7 +33,8 @@
             <div class="spinner-border"></div>
           </td>
           <td class='text-center col-md-4'><%= script.title %></td>
-          <td class='col-md-4'>
+          <td class='col-md-12 text-center'>
+          <div class="btn-group" data-toggle="buttons">
```

Comment:
I don't think we want button groups, because these all redirect somewhere.  I mean, yes they're mutually exclusive buttons, but once you press one the page redirects anyway so there's nothing to disable.

Reply:
I only did that to fix the page to look like this:
![Screenshot from 2023-05-17 15-23-48](https://github.com/OSC/ondemand/assets/4951843/c54066e2-45c9-49b1-b885-2dd92ba158b7)


Instead of this:

![Screenshot from 2023-05-17 15-23-26](https://github.com/OSC/ondemand/assets/4951843/4c0152c7-74a6-41d9-8375-0006be836e94)

Reply:
I can yank the `data-toggle="buttons"` though, that doesn't seem to make any difference and I forget why I even added it now.